psi4/src/psi4/libmints/multipolepotential.h

```diff
@@ -57,7 +57,7 @@ class MultipolePotentialInt : public OneBodyAOInt {
     int max_k_;
```

Comment:
Add the comment about how to know when we can get rid of this, and how to get rid of this, that I've mentioned in a couple other header files.

Reply:
See #2414, as per comments above. I don't think a comment in the code is needed.